kolibri/core/device/api.py

```diff
@@ -107,32 +107,21 @@ def get(self, request, format=None):
         return Response(info)
 
 
-class DeviceLanguageSettingViewset(views.APIView):
+class DeviceSettingsView(views.APIView):
```

Comment:
Rather than mapping these settings into the facility settings above, we could bootstrap our device settings into the base template, as they are used by everyone. 

This works for now though!